docs/data/view.md

```diff
@@ -51,7 +53,7 @@ To delete all image, file, or sensor data respectively, click on **DELETE ALL**
 
 ![Data view with no image selected showing the Delete all button](/data/delete_all.png)
 
-You can also delete data using the [Viam CLI](/fleet/cli/).
+Alternatively, you can use the [`viam data dataset remove` command](/fleet/cli/#data) to remove an image or group of images using the Viam CLI.
```

Comment:
Similarly to the above comment, datasets are different from the data view and aren't affected by this CLI addition, so would keep this whole `view.md` file as-is.